.github/workflows/pull-request.yml

```
@@ -466,3 +466,70 @@
         run: |
           echo "Error: Pull request target branch must be 'develop-postgres'. Please refer PR_GUIDELINES.md"
           exit 1
+          
+  generate-docs:
+    name: Generate Auto Documentation
+    runs-on: ubuntu-latest
+    needs: [Code-Quality-Checks]
+    steps:
+      - uses: actions/checkout@v4
+        with:
```

actionlint (1.7.4)

"with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)